tdrs-backend/tdpservice/users/test/test_api/test_endpoints.py

```diff
@@ -50,10 +50,6 @@ def get_user(self, api_client, user_id):
         """Get a specific user."""
         return api_client.get(f'{self.root_url}{user_id}/')
 
-    def set_profile(self, api_client, profile_data):
-        """Patch a users profile data."""
```

Comment:
@jtimpe not quite sure where this is happening but hitting this endpoint as an approved user with a role yields this:

`ValidationError at /v1/users/set_profile/`
```
<html>
<body>
<!--StartFragment-->
Exception Type: | ValidationError
-- | --
['“set_profile” is not a valid UUID.']

<!--EndFragment-->
</body>
</html>

Reply:
`/v1/users/set_profile` no longer maps to a route, so it tries to hit `/v1/users/{id}` with a value of `'set_profile'` for `'id`. it looks a bit weird but is kind of expected. in an environment where `DJANGO_DEBUG=False` it should just return a standard error message/page, though it might be a 500 because of the exception. we could potentially make it nicer by handling this and returning 404 instead, but i think this is probably happening anyway if you put a non-uuid formatted value in for `/users/{id}` regardless of the `set_profile`